gaia-list.js

```diff
@@ -256,15 +256,6 @@ module.exports = component.register('gaia-list', {
       width: auto;
     }
 
-    /**
-     * Reverse the icons when the document is RTL mode
-     */
-
-    :host-context([dir=rtl]) ::content i:before {
-      transform: scale(-1, 1);
-      text-align: end;
-    }
-
```

Comment:
Let's keep this rule but target it to `[data-icon="right"]` and `[data-icon="right-light"]` so we don't get the wrong icons flipping.